### ISSUE TITLE
The transaction cannot be received when disordering packet and crafting malicious block header before block sync

### ISSUE THREAD

**Issue by fCorleone** [FISCO-BCOS/FISCO-BCOS#3307]
**Describe the bug**
I test FISCO-BCOS 3.1.0 with some chaos strategies. The chaos node may disorder the consensus packets and modify the TxRoot of the blockheader to an invalid value during the block sync. I setup a 10-node group where node 7, 8 and 9 are equipped with these strategies. Then I started the stress testing program and it just stucked.
<img width="1280" alt="屏幕快照 2023-01-05 下午3 24 50" src="https://user-images.githubusercontent.com/22519865/210769397-23c6e5c4-df32-4a9a-9784-2ec66434b12a.png">

The logs are attached.

**To Reproduce**
1. Start a 10-node group.
2. Start the stress testing program with the following command:
```
java -cp 'conf/:lib/*:apps/*' org.fisco.bcos.sdk.demo.perf.PerformanceTransferDMC group0 1 50000 2000
```

**Expected behavior**
The stress testing program should not be stuck.

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - FISCO BCOS Version 3.1.0
 
**Additional context**
Logs:
[logs_0105_consensus.zip](https://github.com/FISCO-BCOS/FISCO-BCOS/files/10351703/logs_0105_consensus.zip)

**Comment by JimmyShi22** [comment]
That's right! The change of TxRoot will not reach consensus if threre are some chaos strategies here. Thus some transaction timeout.
